src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/ActionBarViewModel.cs

```diff
@@ -11,7 +11,8 @@
 namespace Microsoft.CmdPal.UI.ViewModels;
 
 public partial class ActionBarViewModel : ObservableObject,
-    IRecipient<UpdateActionBarMessage>
+    IRecipient<UpdateActionBarMessage>,
+    IRecipient<UpdateActionBarPage>
```

Comment:
Feel like this is bleed over or will conflict with the #199 (some clean-up of 'CurrentPage'), as I think I cleaned this up a bit. We should just be careful merging.

Reply:
Saw this resolved with the merged update from main.